nautobot/docs/user-guide/administration/guides/request-profiling.md

```diff
@@ -10,6 +10,12 @@ Request profiling may be enabled by individual users in their profile within the
 
 Once a user enables request profiling, all subsequent HTTP requests made by that specific user to the system will be logged by django-silk. This setting will persist until the user either logs out or disables the setting in their profile.
 
+### User Setting Notes
+
+- This option will only be usable once the global configuration option `ALLOW_REQUEST_PROFILING` has been enabled (described below).
+
+- *Warning!* This will impact the performance of Nautobot greatly! This should be disabled when not in use.
```

Comment:
Would be good to clarify `This` here - IIRC it's just the user-enabled request profiling, globally enabling it shouldn't have any significant impact if no user has it enabled?

Reply:
Updated with additional clarification that global is not an issue.

Reply:
Did you miss a `git add`? I don't see any change to the text.

Reply:
no, you're right. i used the github direct inline editor for all of this, and attempted two changes via it... herp derp on my part.